clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/core/ClouddriverHostname.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.core;
+
+
+import java.net.InetAddress;
+import java.util.UUID;
+
+public class ClouddriverHostname {
+  public final static String ID = id();
+
+  private static String id() {
+    String hostname;
+    try {
+      hostname = InetAddress.getLocalHost().getHostName();
+    } catch (Exception e) {
+      hostname = "UNKNOWN";
+    }
+
+    return UUID.randomUUID() + "@" + hostname;
```

Comment:
One of @robzienert 's recent blog points introduced me to [ULID](https://github.com/ulid) which I now am a big fan of because it is far easier to read. Sample:

```
ulid() // 01ARZ3NDEKTSV4RRFFQ69G5FAV
```